### PR TITLE
MINOR FIX : CalendarEntry Service Exception.

### PR DESCRIPTION
**Pull request by imjane5**

### CODE REVIEW

File: OtripleS.Web.Api/Models/CalendarEntries/Exceptions/FailedCalendarEntryServiceException.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+using System;
+using Xeptions;
+
+namespace OtripleS.Web.Api.Models.CalendarEntries.Exceptions
+{
+    public class FailedCalendarEntryServiceException : Xeption
+    {
+        public FailedCalendarEntryServiceException(Exception innerexception)
+            : base(message: "Failed calendar entry service error occured,contact support.", innerexception)
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Models/CalendarEntries/Exceptions/FailedCalendarEntryServiceException.cs:14]: ,<space>contact

inner<capital>Exception

File: OtripleS.Web.Api/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentService.Exceptions.cs [hunk 2]
```diff
@@ -69,6 +69,7 @@ public partial class CalendarEntryAttachmentService
             }
             catch (Exception exception)
             {
+
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentService.Exceptions.cs]: Remove this extra line

**Review by ShriHumrudha — APPROVED** [review]

File: OtripleS.Web.Api/Models/CalendarEntries/Exceptions/FailedCalendarEntryServiceException.cs (hunk 1 above)
Comment (imjane5) [reply OtripleS.Web.Api/Models/CalendarEntries/Exceptions/FailedCalendarEntryServiceException.cs:14]: Updated the same

**Review by ShriHumrudha — APPROVED** [review]

File: OtripleS.Web.Api/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentService.Exceptions.cs (hunk 2 above)
Comment (imjane5) [reply OtripleS.Web.Api/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentService.Exceptions.cs]: Removed the extra line

**Pull request merged**